src/main/java/com/exadel/discount/repository/VendorLocationRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.exadel.discount.repository;
+
+import com.exadel.discount.entity.Vendor;
+import com.exadel.discount.entity.VendorLocation;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+@Repository
+public interface VendorLocationRepository extends JpaRepository<VendorLocation, UUID> {
+
+    List<VendorLocation> findByVendor(Vendor vendor);
```

Comment:
Are you actually wanna search by full vendor entity or just specific fk?

Reply:
non actually, edited to search by vendorId